### ISSUE TITLE
Visualization: ignore wypp wrapper

### ISSUE THREAD

**Issue by skogsbaer** [skogsbaer/write-your-python-program#145]
```python
from wypp import *

@record
class Item:
    name: str

item1 = Item('foo')

@record
class Items:
    items: list[Item]

sampleInvoice = Items([item1])
print(sampleInvoice)
```

When visualizing this program, `sampleItems.items` is displayed as class WyppTypeCheck, leading to a very cluttered visualization. We should ignore all type checking wrapper classes.

**Comment by skogsbaer** [comment]
@hannesbraun how would you fix this bug? I would like to try a fix, but need some guiding.

**Comment by hannesbraun** [comment]
Spontaneously, I would probably add an extra case to the else branch in the store method of the Heap class.
However, I currently don't see a reference to the item object. I am probably currently filtering it out for some reason. If you know where it is, you could modify the references in our generated data model and also add the list again to the list of inner values manually. But I'm not sure if this is the best solution.

I can also have a look at this if you'd like. Tomorrow, I should have time again to work on this.

**Comment by skogsbaer** [comment]
The best thing to do is to avoid constructing wrapper altogether when running in visualization mode. I'll take care of this.

**Comment by skogsbaer** [comment]
fixed in #157
